.travis.yml

```diff
@@ -36,11 +36,11 @@ before_install:
 
 env:
   global:
-  - LINT_CHECK=0 TESTS="0" VERSION="saas-17" ODOO_REPO="odoo/odoo"
+  - LINT_CHECK=0 TESTS="0" VERSION="saas-17" ODOO_BRANCH="saas-17-valuation-merge" ODOO_REPO="odoo-dev/odoo"
   - secure: "EG6TofH3CiZKZmSHLDOHsMVSbYYMSXUO2XewjlWGgnqYsFkfpzos5d/0f6Yoty30pJkEj67SA1u9UJk9EDAxJSIpWcho18cmCSIOvaJfge58yhB/YbHUDRBy2rT47ertlrgc1vodfPXHi+Q30O3pxTwlZibGhL6tt694ce39rpE="
 
   matrix:
-  - TESTS="1" INCLUDE="stock,stock_account,sale_stock,stock_landed_costs"
+  - TESTS="1" INCLUDE="avco_tests"
 
 install:
   - git clone --single-branch --depth=1 https://github.com/Vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
```

Comment:
You cant change the line 46 for the follow line:
`- git clone --single-branch --depth=1 https://github.com/vauxoo-dev/maintainer-quality-tools.git -b master-fix-pull-request-no-git-jesuszapata ${HOME}/maintainer-quality-tools`